php/Dockerfile

```diff
@@ -2,9 +2,6 @@ FROM microsoft/iis:windowsservercore
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
-ENV PHP_VERSION 7.1.3

```

Comment:
I would prefer having the version number in the Dockerfile to make it more visible. But otherwise I also only use `microsoft/iis:windowsservercore` and not a specific version for the base layer :-)